MANIFEST.in

```
@@ -1,5 +1,5 @@
 include README.rst CHANGES LICENSE
-include test_responses.py test_matchers.py test_registries.py
+include **/test_*.py
```

I just did a test, we will miss `__init__` file that way

this pattern ensures that all python files within `responses` are included.
@markstory 
I think we expect that all .py under the package are included, right?

```suggestion
recursive-include responses *.py
```

---

> I just did a test, we will miss `__init__` file that way

Correct though I failed to notice as the sdist did contain all the tests and pytest ran all tests successfully in Fedora's build system. I'll add your change in a second.